client/src/ui/track/JobCard.tsx

```diff
@@ -115,9 +145,72 @@ export function JobCard({
             customSx={{
               height: "20px",
               fontSize: "10px",
-              width: "70px", // Adjust width for fit
+              width: "70px",
             }}
           />
+          {/* Display the score if status is OA */}
+          {Status === "Online Assesstment" && currentScore && outOfScore && (
+            <Typography
+              variant="caption"
+              sx={{
+                backgroundColor: "#496FFF",
+                color: "white",
+                borderRadius: "8px",
+                padding: "2px 6px",
+                fontSize: "10px",
+              }}
+            >
+              {currentScore}/{outOfScore}
+            </Typography>
+          )}
+          {/* Display the score if status is OA */}
+          {Status === "Interviewing" && (
```

Comment:
Interviewing button is too big
<img width="310" alt="image" src="https://github.com/user-attachments/assets/c4fdbf87-2cf2-4a6f-99ac-d59c3a07679c">

Reply:
I make the column the bigger.